src/analyses/libraryFunctions.ml

```diff
@@ -167,23 +167,34 @@ let invalidate_actions = [
     "GetSpinlock", readsAll;
     "ReleaseSpinlock", readsAll;
     "atoi", readsAll;             (*safe*)
+    "__builtin_ctz", readsAll;
+    "__builtin_ctzl", readsAll;
+    "__builtin_ctzll", readsAll;
+    "__builtin_clz", readsAll;
     "bzero", writes [1]; (*keep 1*)
     "connect", readsAll;          (*safe*)
     "fclose", readsAll;           (*safe*)
     "fflush", writesAll;          (*unsafe*)
     "fopen", readsAll;            (*safe*)
+    "fdopen", readsAll;            (*safe*)
+    "setvbuf", writes[2];
     "fprintf", writes [1];          (*keep [1]*)
     "__fprintf_chk", writes [1];    (*keep [1]*)
     "fread", writes [1];            (*keep [1]*)
+    "__fread_alias", writes [1];            (*keep [1]*)
+    "__fread_chk", writes [1];            (*keep [1]*)
```

Comment:
I assume that they also might modify the `FILE` pointer they are given (e.g. advancing some cursor?)

Reply:
I changed it such that this is written to, but my suspicion is that we don't really do anything terribly useful with them anyway.

Reply:
>my suspicion is that we don't really do anything terribly useful with them anyway.

Probably.